backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -31,18 +31,48 @@ public class UserService : BaseService<User>, IUserService
         #region Methods
         /// <summary>
         /// Get a page of users from the datasource.
+        /// The filter will allow queries to search for anything that starts with the following properties; DisplayName, FirstName, LastName, Email, Agencies.
         /// </summary>
         /// <param name="page"></param>
         /// <param name="quantity"></param>
-        /// <param name="sort"></param>
         /// <returns></returns>
-        public Paged<User> GetNoTracking(int page = 1, int quantity = 10, string sort = null)
+        public Paged<User> GetNoTracking(int page = 1, int quantity = 10)
```

Comment:
Basic method to page through all users.

Reply:
Could you please add a comment why we are no longer going to support sorting here? thanks

Reply:
The more advanced method is a few lines down.  All this method provides is a simple interface for requesting a page of all users.  The method probably should just be removed, but I don't think it's critical atm.